### PR TITLE
[Issue 5157][docs] Update source and sink --processing-guarantees description 

### LINKED ISSUE #5157: [docs] How does --processing-guarantees affect source connectors?
The command `sources create` accepts a `--processing-guarantees` option. I tried to find out what's the difference between the accepted options but I could not find an answer in the docs.

Did I miss something or does the documentation need a little improvement?

Thank you!

### PR DESCRIPTION
**Pull request by Jennifer88huang**
Fixes #5157

### Motivation
The description for source and sink `--processing-guarantees` is not clear.

### Modifications
Update related description for it.

### CODE REVIEW

File: site2/docs/io-cli.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ $ pulsar-admin sources create options
 |`--name` | The source's name.
 | `--namespace` | The source's namespace.
 | ` --parallelism` | The source's parallelism factor, that is, the number of source instances to run.
-| `--processing-guarantees` | The processing guarantees (aka delivery semantics) applied to the source.<br>Possible Values: ATLEAST_ONCE, ATMOST_ONCE, EFFECTIVELY_ONCE.
+| `--processing-guarantees` | The processing guarantees (also named as delivery semantics) applied to the source. A source connector receives messages from external system and writes messages to a Pulsar topic. The `--processing-guarantees` is use to ensure the processing guarantees for writing messages to the Pulsar topic. <br>The available values are ATLEAST_ONCE, ATMOST_ONCE, EFFECTIVELY_ONCE.
```
Comment (sijia-w) [inline site2/docs/io-cli.md:67]:
```suggestion
| `--processing-guarantees` | The processing guarantees (also named as delivery semantics) applied to the source. A source connector receives messages from external system and writes messages to a Pulsar topic. The `--processing-guarantees` is used to ensure the processing guarantees for writing messages to the Pulsar topic. <br>The available values are ATLEAST_ONCE, ATMOST_ONCE, EFFECTIVELY_ONCE.
```

File: site2/docs/io-cli.md [hunk 2]
```diff
@@ -95,7 +95,7 @@ $ pulsar-admin sources update options
...
```
Comment (sijia-w) [inline site2/docs/io-cli.md:98]:
```suggestion
| `--processing-guarantees` | The processing guarantees (also named as delivery semantics) applied to the source. A source connector receives messages from external system and writes messages to a Pulsar topic. The `--processing-guarantees` is used to ensure the processing guarantees for writing messages to the Pulsar topic. <br>The available values are ATLEAST_ONCE, ATMOST_ONCE, EFFECTIVELY_ONCE.
```

File: site2/docs/io-cli.md [hunk 3]
```diff
@@ -263,7 +263,7 @@ $ pulsar-admin sources localrun options
 |`--name`|The source’s name.|
 |`--namespace`|The source’s namespace.|
 |`--parallelism`|The source’s parallelism factor, that is, the number of source instances to run).|
-|`--processing-guarantees`|The processing guarantees (aka delivery semantics) applied to the source. <br>Available values: ATLEAST_ONCE, ATMOST_ONCE, EFFECTIVELY_ONCE.
+|`--processing-guarantees` | The processing guarantees (also named as delivery semantics) applied to the source. A source connector receives messages from external system and writes messages to a Pulsar topic. The `--processing-guarantees` is use to ensure the processing guarantees for writing messages to the Pulsar topic. <br>The available values are ATLEAST_ONCE, ATMOST_ONCE, EFFECTIVELY_ONCE.
```
Comment (sijia-w) [inline site2/docs/io-cli.md:266]:
```suggestion
|`--processing-guarantees` | The processing guarantees (also named as delivery semantics) applied to the source. A source connector receives messages from external system and writes messages to a Pulsar topic. The `--processing-guarantees` is used to ensure the processing guarantees for writing messages to the Pulsar topic. <br>The available values are ATLEAST_ONCE, ATMOST_ONCE, EFFECTIVELY_ONCE.
```

**Comment by Jennifer88huang** [comment]
@sijia-w thanks for your advice.

**Pull request merged**